index.js

```diff
@@ -29,29 +29,66 @@ module.exports = class GarminNodeApi {
             'consumeServiceTicket' : 'false'
         });
 
-        var response = await axios.post("https://sso.garmin.com/sso/login?" + requestQuery, loginQuery);
+        var response = await axios.post('https://sso.garmin.com/sso/login?' + requestQuery, loginQuery);
         var loginResponse = response.data;
         if(loginResponse.includes('Login Successful')) {
             var ticketRegex = /ticket=([^\"]+)\"/;
             var ticket = loginResponse.match(ticketRegex)[1];
+            console.log('Login OK. Ticket: ' + ticket)
             try {
-                await axios.post('https://connect.garmin.com/post-auth/login?ticket=' + ticket);
-            } catch(e)  {
-                await axios.get(e.response.headers.location);
-            };
+                var url = 'https://connect.garmin.com/modern/?ticket=' + ticket
+                response = await axios.post(url)
+                var modernResponse = response.data;
+                var displayNameRegex = /displayName\\\":\\\"([^\"]+)\\\",/;
```

Comment:
Is this display name stuff necessary here? Looks like maybe it is grabbing the display name out of the html?